samples/meetings-audio-state/nodejs/src/components/configure.jsx

```diff
@@ -0,0 +1,33 @@
+// <copyright file="configure.jsx" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+
+import {  useEffect } from "react";
+import { Text } from "@fluentui/react-northstar";
+import * as microsoftTeams from "@microsoft/teams-js";
+
+// Configure page.
+const Configure = props => {
+    useEffect(() => {
+        microsoftTeams.app.initialize().then(() => {
+            microsoftTeams.app.notifySuccess();
+
+            microsoftTeams.pages.config.registerOnSaveHandler(function (saveEvent) {
+                microsoftTeams.pages.config.setConfig({
+                    entityID: "TeamworkTagsTab",
+                    contentUrl: `${window.location.origin}/myview`,
+                    suggestedTabName: "Teams Tab",
+                    websiteUrl: `${window.location.origin}/myview`,
```

Comment:
Are we using this view or page ?  /myview

Reply:
**myview** is nothing but a separate component in that we are displaying. I will change to a meaningful name.